pkg/server/router.go

```diff
@@ -48,6 +49,9 @@ func (r *Router) Routes(h router.Handlers) http.Handler {
 
 	sbDownloadHandler := supportbundle.NewDownloadHandler(r.scaled, r.options.Namespace)
 	m.Path("/v1/harvester/supportbundles/{bundleName}/download").Methods("GET").Handler(sbDownloadHandler)
+
+	btHealthyHandler := backuptarget.NewHealthyHandler(r.scaled)
+	m.Path("/v1/harvester/backuptarget/healthy").Methods("GET").Handler(btHealthyHandler)
```

Comment:
nit: how about just `health`? noun seems to be more compliant with REST.

Reply:
How about `healthz`?
ref: https://kubernetes.io/docs/reference/using-api/health-checks/

Reply:
Good idea!

Reply:
Updated it. Thank you.